BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -220,6 +221,21 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     state.delegate?.presentBraveRewardsController(controller)
   }
   
+  @objc private func tappedMonthlyTip() {
+    // TODO: Replace with actual values.
+    let options = [
+      BATValue(1),
+      BATValue(5),
+      BATValue(10)
+    ]
+    let optionsVC = OptionsSelectionViewController(options: options, selectedOptionIndex: 1, optionSelected: { [weak self] index in
+      // TODO: save selection and update UI
+      self?.publisherSummaryView.monthlyTipView.batValueView.amountLabel.text = options[safe: index]?.displayString ?? options[0].displayString
```

Comment:
I guess you missed the comment I left on the review before. This should pop back to the wallet as well (see settings options for instance)

Reply:
Oh sorry, didnt see that comment. Updated.